nautobot/extras/templates/extras/graphqlquery_list.html

```diff
@@ -0,0 +1 @@
+{% extends "generic/object_list.html" %}
```

Comment:
Still not sure why this file needed to be added given it was working fine before, but not a critical blocker.